scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -1,5 +1,46 @@
 {% from "variants/utils.html" import tier_cell %}
 
+{% macro variant_transcripts(gene) %}
+<!-- Used in case_report template. Populate a table div with gene transcripts that are canonical, primary (max 5) or disease-associated -->
+  {% set n_primary_txs = namespace(count=0) %}
+  <ul>
+    {% for transcript in gene.transcripts %}
+      <!-- check if it's a primary transcript -->
+      {% set hgnc_prim = namespace(primary=None)%}
+      {% for refseq_id in transcript.refseq_identifiers %}
+        {% if gene.common and refseq_id in gene.common.primary_transcripts %}
+          {% set hgnc_prim.primary = refseq_id %}
+        {% endif %}
+      {% endfor %}
+
+      <!-- increase counter if it has associated refseq -->
+      {% if transcript.refseq_identifiers %}
+        {% set n_primary_txs.count = n_primary_txs.count + 1 %}
+      {% endif %}
+
+        <!-- show transcript info only if it's canonical, disease-associated or primary (print max 5 primary transcripts) -->
+      {% if transcript.refseq_identifiers and n_primary_txs.count <= 5 or transcript.is_canonical or transcript.is_disease_associated or hgnc_prim.primary %}
+        <li>{{transcript.transcript_id}}, refseq:[{{ transcript.refseq_identifiers|join(", ") or "-" }}], {{ (transcript.coding_sequence_name or '')|truncate(20, True) }},  {{ (transcript.protein_sequence_name or '')|url_decode }}
+          {% if transcript.is_canonical %}
+            <span class="badge badge-pill badge-secondary" title="canonical">C</span>
+          {% endif %}
+          {% if transcript.is_disease_associated %}
+            <span class="badge badge-pill badge-danger" title="disease_associated">D</span>
+          {% endif %}
+          {% if hgnc_prim.primary %}
```

Comment:
What do you think, any chance of moving this to the RefSeq native one? I guess its the version number that gets in the way?

Reply:
I'll try!

Reply:
I don't know if you meant like this, but I've simplified the code in the macro, check the last 2 commits

Reply:
Thanks!